src/Common/src/Interop/Unix/System.Net.Http.Native/Interop.Initialization.cs

```diff
@@ -23,9 +23,11 @@ internal static class HttpInitializer
     {
         static HttpInitializer()
         {
+#if !SYSNETHTTP_NO_OPENSSL
```

Comment:
Correct.  I've been testing on my system with having deleted System.Security.Cryptography.Native.OpenSsl.dylib, so if anything calls into it I get an exception.